src/App.scss

```diff
@@ -239,6 +239,52 @@ body {
     justify-content: center;
 }
 
+
+.App__ShareButton {
+    background-color: #33717F;
+    border: 0;
+    border-radius: 0.2rem;
+    color: #FFFFFF;
+    font-weight: 400;
+    padding: 0.5rem 2rem;
+    width: 10rem;
+
+    display:flex;
+    grid-template-columns: 30% 70%;
+    column-gap: 0.5rem;
+    align-items: center;
+
+    &:active {
+        background-color: #ED972C;
+    }
+
+    &:focus {
+        box-shadow:
+            0 0 0 0.1rem #D5D8DD,
+            0 0 0 $c2lc-focus-indicator-width black;
```

Comment:
Probably make sense to use #1E1E1E here.

Reply:
Done.